GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
+
+    guard let dict = plist as? Dictionary<String, String> else {
+      throw StringsFileParserError.InvalidFormat
+    }
+
     var encoding: NSStringEncoding = NSUTF16StringEncoding
     let fileContent = try NSString(contentsOfFile: path, usedEncoding: &encoding)
+
+    func extractKey(line: String) -> String? {
```

Comment:
Ah, good point.

Not sure what's best. I would consider sorting keys alphabetically upon generation (= don't sort the dictionary keys, but sort the array of `Entry` once it's build or when building the StencilContext from it), that doesn't seem too bad.

Sure that would mean that the original order in the b`.strings` file isn't kept but that would avoid parsing the file twice, once as a PLIST and once as text, so… 🤔